components/brave_wallet/browser/eth_gas_utils.cc

```diff
@@ -16,6 +16,33 @@ namespace brave_wallet {
 
 namespace eth {
 
+absl::optional<uint256_t> ScaleBaseFeePerGas(const std::string value) {
```

Comment:
const std::string&

Reply:
Pls also add a comment that describes this function.Thanks!

Reply:
Fixed both remarks here: https://github.com/brave/brave-core/pull/18707/commits/697d3a9c27d18649802da7481402aad605ab19f3